### DISCUSSION TITLE
Can't install extensions with code-server on Termux

### DISCUSSION THREAD

**Discussion by Clem0803** [coder/code-server#4223] [Q&A]
When I try to install extensions, I have everytime the same error : Can't install 'ms-python.python' extension because it is not compatible with the current version of VS Code (version 1.60.0).
I run code server version 3.12.0 on Android with Termux and I use Brave as browser but I try Chrome and there is no difference.
![Screenshot_20210922-184327_Termux](https://user-images.githubusercontent.com/48407049/134386186-c520aeb7-8eb4-4250-820b-7797e1059396.jpg)

**Comment by jsjoeio** [comment]
Hmm...I think this is specific to that extension. Can you try this?

```shell
SERVICE_URL=https://open-vsx.org/vscode/gallery ITEM_URL=https://open-vsx.org/vscode/item code-server --install-extension  ms-python.python
```

**Reply by Clem0803** [reply]
This is not specific, I try with other extension and there is the same error. And I try your command but it doesn't work
![Screenshot_20210922-195048_Termux](https://user-images.githubusercontent.com/48407049/134395684-27225a74-87d2-4666-b585-94d8545bdfeb.jpg)
![Screenshot_20210922-194354_Termux](https://user-images.githubusercontent.com/48407049/134395691-5bcae658-c07b-453e-b355-0191ade644a2.jpg)

**Reply by jsjoeio** [reply]
Hmm... it's all the same error though (not compatible with current version of VS Code 🤔 Sorry, typo in my command.

Updated it. Can you try again?

**Reply by Clem0803** [reply]
It's different but same error again
![Screenshot_20210923-120520_Termux](https://user-images.githubusercontent.com/48407049/134489396-554f87eb-3b6c-4e0b-8d5a-52aec14757ed.jpg)

**Reply by jsjoeio** [reply]
This is so weird 🤔 Can you double check the VS Code version? Run `code-server --version --json` and then paste the logs here?

I just installed the Python version with 3.12.0 on my mac and it worked.

**Reply by Clem0803** [reply]
I got this
`~ $ code-server --version --json
{
  codeServer: '3.12.0',
  commit: '4cd55f94c0a72f05c18cea070e10b969996614d2',
  vscode: '1.60.0'
}
~ $`

**Reply by jsjoeio** [reply]
Yeah, that all looks right. Hmm..this is odd. Looking at the Python extension, it says the requirements are VS Code version 1.59.0 or above 🤔 https://github.com/microsoft/vscode-python/blob/main/package.json#L39-L41

We'll have to further investigate. Do you mind opening an issue and linking to this discussion?

**Reply by Clem0803** [reply]
Issue create but this is very weird cause few month ago I have got install code server on the same tablet and I didn't have this error so it's maybe the version of code server ? Is there a way to intall a previous version of code server ?

**Reply by jsjoeio** [reply]
Thank you!

Assuming you installed with yarn or npm, you can do 
```shell
yarn global add code-server@3.11.0
```

to install a previous version. See all versions [here](https://www.npmjs.com/package/code-server)

**Reply by Clem0803** [reply]
So I installed a previous version and it works. I can install extensions without error so the version 3.12.0 doesn't work for me but I don't know why.
Thank you for your help
